test/fixtures/relay.json

```diff
@@ -109,5 +109,14 @@
         "values": ["allowed.header"]
       }
     ]
+  },
+  {
+    "//": "token auth route",
+    "method": "GET",
+    "path": "/bearer-auth",
+    "auth": {
+      "scheme": "bearer",
+      "bearer": "1234"
```

Comment:
In Bitbucket server's case, how will this work? Wonder whether the documentation on this is enough.

Assuming they'll need to manually add this to every accept.json item
```
    "auth": {
      "scheme": "bearer",
      "bearer": "${BITBUCKET_PERSONAL_ACCESS_TOKEN}"
```

Reply:
yup, that's pretty much what needs to b done, updating all the docs is out of scope for me tbh as im doing this as part of oncall... as i mentioned to tom...

Reply:
Discussed in Slack briefly, but for posterity:
@carlos-snyk is completely right that the whole whitelisting will need to be updating. I'm interested to know if this is something we have documented anywhere as it seems ripe for errors. Something we should look at internally in Snyk to determine a responsible approach.